conf/modules.config

```diff
@@ -0,0 +1,137 @@
+/*
+========================================================================================
+    Config file for defining DSL2 per module options and publishing paths
+========================================================================================
+    Available keys to override module options:
+        ext.args            = Additional arguments appended to command in module.
+        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix          = File name prefix for output files.
+----------------------------------------------------------------------------------------
+*/
+
+process {
+
```

Comment:
You could add here a default. Probably like publishing by default [https://github.com/nf-core/rnaseq/conf/modules.config](https://github.com/nf-core/rnaseq/blob/646723c70f04ee6d66391758b02822d4f0fe2966/conf/modules.config#L18-L22) or not [https://github.com/nf-core/ampliseq/conf/modules.config](https://github.com/nf-core/ampliseq/blob/f1abc2010e849903f429a60e130da92b4ad2f8f8/conf/modules.config#L26-L29)